### PR TITLE
Fix visibility of org avatars

### LINKED ISSUE #17778: Organization avatar is not displayed in headers
### Gitea Version

1.16.0+dev-572-g7743f13be

### Operating System

_No response_

### Browser Version

Any

### Can you reproduce the bug on the Gitea demo site?

Yes

### Description

Organization custom avatars are no longer displayed on header. Can be reproduced on try.gitea.io when browsing any organization with avatar set. The avatar is still displayed in few places, most notably on explore page.

The issue occurred between 1.16.0+dev-496-g47448083a and 1.16.0+dev-572-g7743f13be

### Screenshots

firefox_2021-11-23_12-30-51

---

firefox_2021-11-23_12-30-58

firefox_2021-11-23_12-31-04

---

firefox_2021-11-23_12-31-11

### PR DESCRIPTION
**Pull request by qwerty287**
Views org avatars again. 

Fixes #17778

### CODE REVIEW

**Review by zeripath — APPROVED** [review]

File: modules/templates/helper.go [hunk 1]
```diff
@@ -565,6 +565,12 @@ func Avatar(item interface{}, others ...interface{}) template.HTML {
 			return AvatarHTML(src, size, class, user.DisplayName())
 		}
 	}
+	if user, ok := item.(*models.Organization); ok {
+		src := (*models.User)(user).AvatarLinkWithSize(size * avatars.AvatarRenderedSizeFactor)
```
Comment (Gusted) [inline modules/templates/helper.go]:
```suggestion
		src := org.AsUser().AvatarLinkWithSize(size * avatars.AvatarRenderedSizeFactor)
```

File: modules/templates/helper.go [hunk 2]
```diff
@@ -565,6 +565,12 @@ func Avatar(item interface{}, others ...interface{}) template.HTML {
 			return AvatarHTML(src, size, class, user.DisplayName())
 		}
 	}
+	if user, ok := item.(*models.Organization); ok {
```
Comment (Gusted) [inline modules/templates/helper.go]: I don't think it should be called "user", let's call it "org".
```suggestion
	if org, ok := item.(*models.Organization); ok {
```

File: modules/templates/helper.go [hunk 3]
```diff
@@ -565,6 +565,12 @@ func Avatar(item interface{}, others ...interface{}) template.HTML {
...
+		src := (*models.User)(user).AvatarLinkWithSize(size * avatars.AvatarRenderedSizeFactor)
+		if src != "" {
+			return AvatarHTML(src, size, class, user.DisplayName())
```
Comment (Gusted) [inline modules/templates/helper.go]:
```suggestion
			return AvatarHTML(src, size, class, org.DisplayName())
```

**Comment by lunny** [comment]
I think it's better
```
switch t := item.(type) {
case *models.Organization:
default:
}
```

**Comment by lunny** [comment]
@Gusted done.

**Review by Gusted — APPROVED** [review]

**Pull request merged**